libpod/runtime.go

```diff
@@ -374,6 +391,20 @@ func NewRuntime(options ...RuntimeOption) (runtime *Runtime, err error) {
 			return nil, errors.Wrapf(err, "cannot set XDG_RUNTIME_DIR")
 		}
 
+	}
+
+	if userConfigPath != "" {
+		configPath = rootlessConfigPath
```

Comment:
Shouldn't this be userConfigPath not rootlessConfigPath?

Reply:
yes, sorry.